integration-libs/opf/checkout/occ/adapters/occ-opf.adapter.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Injectable, inject } from '@angular/core';
+import {
+  ConverterService,
+  LoggerService,
+  normalizeHttpError,
+} from '@spartacus/core';
+import {
+  OPF_CC_OTP_KEY,
+  OPF_CC_PUBLIC_KEY,
+  OpfConfig,
+} from '@spartacus/opf/base/root';
+import {
+  OPF_PAYMENT_CONFIG_SERIALIZER,
+  OpfAdapter,
+  OpfEndpointsService,
+} from '@spartacus/opf/checkout/core';
+import {
+  PaymentInitiationConfig,
+  PaymentSessionData,
+} from '@spartacus/opf/checkout/root';
+
+import { Observable, throwError } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+
+@Injectable()
+export class OccOpfAdapter implements OpfAdapter {
```

Comment:
[MEDIUM]

To general names. Those classes are strictly related to the checkout flow, but they both names are lacking the word 'Checkout'. Let's rename them to `OccOpfCheckoutAdapter` and `OpfCheckoutAdapter`.

Reply:
Checkout adapter responsibility was moved around other sub-libs.